data/blog/tclip-updates-092023.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: tclip updates for September 2023
+date: 2023-09-19
+authors: ['xe']
+tags:
+  - tclip
+  - tsnet
+summary: 'tclip now has a new theme, org-mode rendering, and runs without CGo.'
+images: ['/images/tclip-updates-092023/social.png']
+---
+
+Hey everyone! We've made a bunch of updates to tclip, and we're excited to share them with you. We hope this will help you get the most out of tclip and give you inspiration to make your own addon tools to do whatever it is you want.
+
+[tclip](https://github.com/tailscale-dev/tclip) is a little addon service that gives everyone on your tailnet a place to share small snippets of text, code, log messages, or anything that can fit into a text box. It's like [GitHub Gists](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists) but private on your tailnet.
+
+You can read more about tclip and how to install it on GitHub at [tailscale-dev/tclip](https://github.com/tailscale-dev/tclip). I also wrote more about tclip in [the previous tclip article](/blog/tclip).
+
+This release has a few features and changes that were contributed by the community. Thank you all for helping us!
+
+With huge thanks to:
+
+- [@gmemstr](https://github.com/gmemstr)
+- [@imeyer](https://github.com/imeyer)
+- [@jolheiser](https://github.com/jolheiser)
+
```

Comment:
and then like:

In this post, we want to show off some of the cool new changes to tclip in particular, but also to give you some inspiration and maybe some pointers if you're looking to make your own tailnet add-on tools.